#def sumar(a, b):
#    return a+b

s=sumar(1)
print(s)
#Traceback (most recent call last):
#  File "fun_call.py", line 4, in <module>
#    s=sumar(1)
#TypeError: sumar() takes exactly 2 arguments (1 given)

def sumar(a, b):
    return a+b
#Traceback (most recent call last):
#  File "fun_call.py", line 4, in <module>
#    s=sumar(1)
#NameError: name 'sumar' is not defined
